apps/web/src/components/CookieBanner/CookieBanner.tsx

```diff
@@ -0,0 +1,163 @@
+        if (view === CookieBannerView.Selection && isCookieSet) {
+          setView(CookieBannerView.Hidden)
+        }
+      }
+    }
+    router.events.on('routeChangeComplete', handleRouteChange)
+    return () => {
+      router.events.off('routeChangeComplete', handleRouteChange)
+    }
+  }, [router, view])
+
+  useEffect(() => {
+    const isCookieSet = Boolean(getCookie(SE_GDPR_COOKIE_NAME))
+
+    // Set initial visibility on client
+    if (view === CookieBannerView.Hidden && !isCookieSet) {
+      setView(CookieBannerView.Selection)
+    }
+
+    // Set focus on confirmation screen
+    if (view === CookieBannerView.Accepted || view === CookieBannerView.Rejected) {
+      regionRef.current?.focus()
+    }
+
+    // Set focus after clicking "change settings"
+    if (view === CookieBannerView.Selection && isCookieSet) {
+      regionRef.current?.focus()
+    }
+  }, [view])
+
+  const handleAccept: MouseEventHandler = () => {
+    setView(CookieBannerView.Accepted)
+    setCookie(SE_GDPR_COOKIE_NAME, SE_GDPR_COOKIE_ACCEPT_VALUE, { expires: getGDPRCookieExpiryDate(), secure: true })
+    window.gtag('consent', 'update', {
+      ad_storage: 'granted',
+      analytics_storage: 'granted',
+      ad_user_data: 'granted',
+      ad_personalization: 'granted',
+    })
+    document.dispatchEvent(new CustomEvent(COOKIE_SETTINGS_CHANGE_EVENT, { detail: 1 }))
+  }
+
+  const handleReject: MouseEventHandler = () => {
+    setView(CookieBannerView.Rejected)
+    setCookie(SE_GDPR_COOKIE_NAME, SE_GDPR_COOKIE_REJECT_VALUE, { expires: getGDPRCookieExpiryDate(), secure: true })
+    window.gtag('consent', 'update', {
+      ad_storage: 'denied',
+      analytics_storage: 'denied',
+      ad_user_data: 'denied',
+      ad_personalization: 'denied',
+    })
+    document.dispatchEvent(new CustomEvent(COOKIE_SETTINGS_CHANGE_EVENT, { detail: 0 }))
+  }
+
+  const handleHide: MouseEventHandler = () => {
+    setView(CookieBannerView.Hidden)
+  }
+
+  const renderSelection = () => {
+    return (
+      <div className="govuk-cookie-banner__message govuk-width-container">
+        <div className="govuk-grid-row">
+          <div className="govuk-grid-column-two-thirds">
+            <h2 className="govuk-cookie-banner__heading govuk-heading-m">Cookies on Sponsor Engagement</h2>
+            <div className="govuk-cookie-banner__content govuk-body focus:outline-0" ref={regionRef} tabIndex={-1}>
+              <p>We may use some essential cookies to make this service work.</p>
+              <p>
+                We’d like to use additional cookies so we can remember your settings, understand how people use the
+                Sponsor Engagement Tool and make improvements.
+              </p>
+            </div>
+          </div>
+        </div>
+
+        <div className="govuk-button-group mb-0">
+          <button className="govuk-button" name="cookies" onClick={handleAccept} type="button" value="accept">
```

Comment:
we do not use the value in these buttons, either remove or alter your onClick for both buttons